python/requirements.txt

```diff
@@ -39,6 +39,7 @@ tensorboardX >= 1.9
 uvicorn
 dataclasses; python_version < '3.7'
 starlette
+ray-cpp
```

Comment:
don't add it here. this is for setting up the surrounding requirements

Reply:
”ray-cpp“ package is a requirement of "ray" package in the way of "pip install ray[cpp]"?

Reply:
this requirements.txt is not used to specify dependency. this file is used only to setup CI infra environment.

Reply:
so in order to CI to pass, you will need to remove this line

Reply:
Got it!